dvc/output/base.py

```diff
@@ -53,6 +53,15 @@ def __init__(self, match):
         super().__init__(f"Path '{match.file}' is ignored by\n{lines}")
 
 
+class OutputStoreCacheIncompatible(DvcException):
+    def __init__(self, path):
+        msg = f"""Incompatible options for output '{path}'.
+    The dvc file specifies an output with both "cache == False" and
```

Comment:
```suggestion
    The .dvc file specifies an output with both "cache == False" and
```

Reply:
Also maybe use `cache: false` and `store: true` (in back quotes) since that's how they're shown in .dvc files.